Makefile

```diff
@@ -0,0 +1,27 @@
+MSBUILD = xbuild
+MDOC = mdoc/bin/Release/mdoc.exe
```

Comment:
Opinions may vary, but I'd suggest a *toplevel* `bin` directory, not a *per-assembly* `bin` directory.

Maybe I'm the crazy one, but this is what I do in Java.Interop and xamarin-android; all build artifacts are placed in `$topdir/bin`.